.github/workflows/go.yml

```diff
@@ -11,8 +11,11 @@ jobs:
         go-version: 1.13
       id: go
 
-    - name: Check out code into the Go module directory
+    - name: Check out code
       uses: actions/checkout@v1
 
     - name: Build
       run: make all
+
+    - name: Run golangci-lint
+      uses: actions-contrib/golangci-lint@v1
```

Comment:
is the PR still work in progress? this step is failing.

Reply:
This is partly WIP. I think the issue is with go 1.13. Wait until #34 is merged to lower the amount of rebase/merge conflict handling.

Reply:
fixed. PTAL